### ISSUE TITLE
[JENKINS-53462] Jenkins websites use non-trusted 'submit' event to start form submission when current browser is Firefox

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-23#8652]
<p>HTML spec [<span class="error">&#91;1&#93;</span>|<a href="https://w3c.github.io/uievents/#trusted-events" class="external-link" target="_blank" rel="nofollow noopener">https://w3c.github.io/uievents/#trusted-events</a>] says "Most untrusted events will not trigger default actions, with the exception of the click event.". Now Firefox doesn't comply with the spec. When I try to fix the bug [<span class="error">&#91;2&#93;</span>|<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630</a>], a regression has happened on all Jenkins websites. Users can't login Jenkins websites with Firefox anymore. After some experiments, it seems the Jenkins websites detect the browser's user agent and use untrusted 'submit' event to start form submission when the current browser is Firefox. Changing the UA of Chrome to the same string as Firefox also block the form submission.</p>

<p> </p>

<p>The steps I used to reproduce this problem</p>

<p>On Chrome</p>
<ol>
	<li>Change UA to the same string as Firefox</li>
	<li>Navigate <a href="https://jenkins.qa.ubuntu.com/" class="external-link" target="_blank" rel="nofollow noopener">https://jenkins.qa.ubuntu.com/</a></li>
	<li>Click login</li>
	<li>Enter username/password and press 'log in' button</li>
	<li>Nothing happened</li>
</ol>

<p>Expectation</p>

<p>Don't use untrusted events to start form submission on Jenkins websites.</p>

<p> </p>

<p><span class="error">&#91;1&#93;</span> <a href="https://w3c.github.io/uievents/#trusted-events" class="external-link" target="_blank" rel="nofollow noopener">https://w3c.github.io/uievents/#trusted-events</a></p>

<p><span class="error">&#91;2&#93;</span> <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630</a></p>

<p> </p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=iamstone">iamstone</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462" target="_blank">Jenkins websites use non-trusted 'submit' event to start form submission when current browser is Firefox</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Major
<li><b>component(s)</b>: core
<li><b>label(s)</b>: 2.164.3-fixed
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2021-04-20T13:29:16+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 9
<li><b>imported</b>: 2025-11-23
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>HTML spec [<span class="error">&#91;1&#93;</span>|<a href="https://w3c.github.io/uievents/#trusted-events" class="external-link" target="_blank" rel="nofollow noopener">https://w3c.github.io/uievents/#trusted-events</a>] says "Most untrusted events will not trigger default actions, with the exception of the click event.". Now Firefox doesn't comply with the spec. When I try to fix the bug [<span class="error">&#91;2&#93;</span>|<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630</a>], a regression has happened on all Jenkins websites. Users can't login Jenkins websites with Firefox anymore. After some experiments, it seems the Jenkins websites detect the browser's user agent and use untrusted 'submit' event to start form submission when the current browser is Firefox. Changing the UA of Chrome to the same string as Firefox also block the form submission.</p>

<p> </p>

<p>The steps I used to reproduce this problem</p>

<p>On Chrome</p>
<ol>
	<li>Change UA to the same string as Firefox</li>
	<li>Navigate <a href="https://jenkins.qa.ubuntu.com/" class="external-link" target="_blank" rel="nofollow noopener">https://jenkins.qa.ubuntu.com/</a></li>
	<li>Click login</li>
	<li>Enter username/password and press 'log in' button</li>
	<li>Nothing happened</li>
</ol>

<p>Expectation</p>

<p>Don't use untrusted events to start form submission on Jenkins websites.</p>

<p> </p>

<p><span class="error">&#91;1&#93;</span> <a href="https://w3c.github.io/uievents/#trusted-events" class="external-link" target="_blank" rel="nofollow noopener">https://w3c.github.io/uievents/#trusted-events</a></p>

<p><span class="error">&#91;2&#93;</span> <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630</a></p>

<p> </p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>classic login form (before 2.128), regular &quot;Save&quot; form submission buttons on the classic UI</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/rtyler.png" title="rtyler's avatar" /> rtyler:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=325349&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-325349">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>The WEBSITE project is for jenkins.io and those sorts of issues.</p>

<p> </p>

<p>That said, jenkins.qa.ubuntu.com is running a version of Jenkins more than two years old at this point.</p>
</pre>
</details>

</ul>
</details>

<p>The WEBSITE project is for jenkins.io and those sorts of issues.</p>

<p> </p>

<p>That said, jenkins.qa.ubuntu.com is running a version of Jenkins more than two years old at this point.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/oleg_nenashev.png" title="oleg_nenashev's avatar" /> oleg_nenashev:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=328074&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-328074">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>FYI <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, seems to be a security-related matter</p>
</pre>
</details>

</ul>
</details>

<p>FYI <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, seems to be a security-related matter</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/top12345tw.svg" title="top12345tw's avatar" /> top12345tw:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=345846&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345846">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hi, I could not reproduce this problem in the weekly version of Jenkins, but I still could reproduce it on LTS version. Should we reopen this issue for LTS version? Thank you.</p>
</pre>
</details>

</ul>
</details>

<p>Hi, I could not reproduce this problem in the weekly version of Jenkins, but I still could reproduce it on LTS version. Should we reopen this issue for LTS version? Thank you.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=348012&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-348012">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>There was a refactoring of the login page which may have changed this in <a href="#v2.128]" target="_blank" rel="noopener">version 2.128</a> from June 18th this year?</p>

<p> </p>

<p>Jira issue: <a href="https://issues.jenkins-ci.org/browse/JENKINS-50447" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/JENKINS-50447</a></p>

<p>Github PR: <a href="https://github.com/jenkinsci/jenkins/pull/3380" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/3380</a></p>

<p> </p>
</pre>
</details>

</ul>
</details>

<p>There was a refactoring of the login page which may have changed this in <a href="#v2.128]" target="_blank" rel="noopener">version 2.128</a> from June 18th this year?</p>

<p> </p>

<p>Jira issue: <a href="https://issues.jenkins-ci.org/browse/JENKINS-50447" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/JENKINS-50447</a></p>

<p>Github PR: <a href="https://github.com/jenkinsci/jenkins/pull/3380" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/3380</a></p>

<p> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=348667&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-348667">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=oleg_nenashev" class="user-hover" rel="oleg_nenashev">oleg_nenashev</a>, <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, if the fix cannot be integrated in the current LTS release, is there already an ETA when the next major LTS release will happen? Thanks.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=oleg_nenashev" class="user-hover" rel="oleg_nenashev">oleg_nenashev</a>, <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, if the fix cannot be integrated in the current LTS release, is there already an ETA when the next major LTS release will happen? Thanks.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=348673&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-348673">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I cannot, and could never, despite Firefox being my default browser, reproduce the issue on any Jenkins instance, and it is unclear that the issue as reported actually exists in current releases of Jenkins. More information is needed before we even consider this to be an open bug.</p>
</pre>
</details>

</ul>
</details>

<p>I cannot, and could never, despite Firefox being my default browser, reproduce the issue on any Jenkins instance, and it is unclear that the issue as reported actually exists in current releases of Jenkins. More information is needed before we even consider this to be an open bug.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=348679&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-348679">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>OK. The bug currently in Firefox allows the login to happen, which is why I never encountered the described problem. Once the bug in Firefox is fixed, it will no longer be able to use the login form. Now I got it.</p>

<p>Would be interesting to know whether simply giving Firefox the 'Chrome behavior' (if any), i.e. modifying or removing browser detection, would make this work. That should be fairly straightforward, I hope.</p>

<p>I'll move this to the correct Jira project.</p>
</pre>
</details>

</ul>
</details>

<p>OK. The bug currently in Firefox allows the login to happen, which is why I never encountered the described problem. Once the bug in Firefox is fixed, it will no longer be able to use the login form. Now I got it.</p>

<p>Would be interesting to know whether simply giving Firefox the 'Chrome behavior' (if any), i.e. modifying or removing browser detection, would make this work. That should be fairly straightforward, I hope.</p>

<p>I'll move this to the correct Jira project.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/tscherler.png" title="tscherler's avatar" /> tscherler:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351029&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351029">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://github.com/jenkinsci/jenkins/pull/3689" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/3689</a> the hack version</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://github.com/jenkinsci/jenkins/pull/3689" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/3689</a> the hack version</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/tscherler.png" title="tscherler's avatar" /> tscherler:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351597&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351597">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> <a href="https://youtu.be/o6bw5KEIqik" class="external-link" target="_blank" rel="nofollow noopener">https://youtu.be/o6bw5KEIqik</a> AFAIC tell it works 
and debug mode: <a href="https://youtu.be/321YMNNozcE" class="external-link" target="_blank" rel="nofollow noopener">https://youtu.be/321YMNNozcE</a></p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> <a href="https://youtu.be/o6bw5KEIqik" class="external-link" target="_blank" rel="nofollow noopener">https://youtu.be/o6bw5KEIqik</a> AFAIC tell it works <br/>
and debug mode: <a href="https://youtu.be/321YMNNozcE" class="external-link" target="_blank" rel="nofollow noopener">https://youtu.be/321YMNNozcE</a></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/tscherler.png" title="tscherler's avatar" /> tscherler:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351649&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351649">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://youtu.be/oT7orP-FdMs" class="external-link" target="_blank" rel="nofollow noopener">https://youtu.be/oT7orP-FdMs</a> I just tested on master <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> and it shows the issue description. I am pretty confident that the PR fixes the issue (not by winning a beauty competition, but...) <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/wink.png" height="16" width="16" align="absmiddle" alt="" border="0"/> </p>
</pre>
</details>

</ul>
</details>

<p><a href="https://youtu.be/oT7orP-FdMs" class="external-link" target="_blank" rel="nofollow noopener">https://youtu.be/oT7orP-FdMs</a> I just tested on master <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> and it shows the issue description. I am pretty confident that the PR fixes the issue (not by winning a beauty competition, but...) <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/wink.png" height="16" width="16" align="absmiddle" alt="" border="0"/> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351663&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351663">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Wonderful. In that case I assume you want to finish up the patch now? Is there something else you would need help with?</p>

<p>My remaining question would be if we could also get this into a LTS release once it landed on master.</p>

<p> </p>
</pre>
</details>

</ul>
</details>

<p>Wonderful. In that case I assume you want to finish up the patch now? Is there something else you would need help with?</p>

<p>My remaining question would be if we could also get this into a LTS release once it landed on master.</p>

<p> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/tscherler.png" title="tscherler's avatar" /> tscherler:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351664&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351664">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>to your last question I guess <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a> or <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=rtyler" class="user-hover" rel="rtyler">rtyler</a> can make this happen. <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/wink.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

<p>The patch just needs some approves and it is done. </p>

<p>BTW I just verified that other parts of jenkins will work as expected and I left a comment in the FF bugzilla to link to the pr as fix when people run into the issue on older versions of jenkins.</p>

<p>HTH and thanks for all your hard work at mozilla!!!</p>
</pre>
</details>

</ul>
</details>

<p>to your last question I guess <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a> or <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=rtyler" class="user-hover" rel="rtyler">rtyler</a> can make this happen. <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/wink.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

<p>The patch just needs some approves and it is done. </p>

<p>BTW I just verified that other parts of jenkins will work as expected and I left a comment in the FF bugzilla to link to the pr as fix when people run into the issue on older versions of jenkins.</p>

<p>HTH and thanks for all your hard work at mozilla!!!</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/tscherler.png" title="tscherler's avatar" /> tscherler:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351666&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351666">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>As final explanation why FF stopped working is that since we did not provide a callback to the onClick, FF correctly stops propagating and all other browser are wrong to propagate the event to a submit.</p>

<p>We fixed the f:submit however the problem may occur in other parts but the solution will always be to provide a default fallback for onClick.</p>
</pre>
</details>

</ul>
</details>

<p>As final explanation why FF stopped working is that since we did not provide a callback to the onClick, FF correctly stops propagating and all other browser are wrong to propagate the event to a submit.</p>

<p>We fixed the f:submit however the problem may occur in other parts but the solution will always be to provide a default fallback for onClick.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351786&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351786">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<blockquote><p>My remaining question would be if we could also get this into a LTS release once it landed on master.</p></blockquote>
<p>Doubtful, the RC for the next LTS is due next Wednesday and we generally require changes to soak for at least two releases (~10 days), preferably three, before backporting.</p>

<p>Should be in the early December LTS though.</p>
</pre>
</details>

</ul>
</details>

<blockquote><p>My remaining question would be if we could also get this into a LTS release once it landed on master.</p></blockquote>
<p>Doubtful, the RC for the next LTS is due next Wednesday and we generally require changes to soak for at least two releases (~10 days), preferably three, before backporting.</p>

<p>Should be in the early December LTS though.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=351815&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-351815">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>That's great to hear that you are considering it for a LTS release even it's not the upcoming one. Thanks a lot!</p>
</pre>
</details>

</ul>
</details>

<p>That's great to hear that you are considering it for a LTS release even it's not the upcoming one. Thanks a lot!</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=352772&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-352772">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, I simply want to report back that one of our engineers who works a lot with Jenkins just checked the latest release of Jenkins and it indeed seems to work all fine. Having it in the next LTS release would be fantastic! Thanks!</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, I simply want to report back that one of our engineers who works a lot with Jenkins just checked the latest release of Jenkins and it indeed seems to work all fine. Having it in the next LTS release would be fantastic! Thanks!</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=364824&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-364824">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>2.164.3, scheduled for May 8, the earliest, if accepted as a backport. The next .1, scheduled for early June, otherwise. Assuming of course we don't cause regressions again.</p>
</pre>
</details>

</ul>
</details>

<p>2.164.3, scheduled for May 8, the earliest, if accepted as a backport. The next .1, scheduled for early June, otherwise. Assuming of course we don't cause regressions again.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=364872&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-364872">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>FWIW if someone wants to test this with a patched Firefox build, <tt>docker run -p 8080:8080 jenkins/jenkins:2.173</tt> and then go to <tt><a href="http://localhost:8080" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:8080</a></tt> – should be fairly straightforward to get through the wizard, then e.g. <em>Manage Jenkins » Configure System</em> to go to a form with a formerly broken submit button ("Save").</p>
</pre>
</details>

</ul>
</details>

<p>FWIW if someone wants to test this with a patched Firefox build, <tt>docker run -p 8080:8080 jenkins/jenkins:2.173</tt> and then go to <tt><a href="http://localhost:8080" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:8080</a></tt> – should be fairly straightforward to get through the wizard, then e.g. <em>Manage Jenkins » Configure System</em> to go to a form with a formerly broken submit button ("Save").</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=364884&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-364884">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I won't have the time for testing it, but I requested a new Firefox build with the patch on <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630</a> applied. Currently the patch doesn't apply anymore. Once we have a build I will happily post a link to it.</p>
</pre>
</details>

</ul>
</details>

<p>I won't have the time for testing it, but I requested a new Firefox build with the patch on <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630</a> applied. Currently the patch doesn't apply anymore. Once we have a build I will happily post a link to it.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/olivergondza.png" title="olivergondza's avatar" /> olivergondza:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365299&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365299">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Can someone verify the patch? I have failed to reproduce the problem using <tt>firefox-nightly-68.0a1.20190422-1</tt> and <tt>jenkins/jenkins:2.172</tt>. Both logging in as admin and saving global config page works for me.</p>
</pre>
</details>

</ul>
</details>

<p>Can someone verify the patch? I have failed to reproduce the problem using <tt>firefox-nightly-68.0a1.20190422-1</tt> and <tt>jenkins/jenkins:2.172</tt>. Both logging in as admin and saving global config page works for me.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365301&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365301">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=olivergondza" class="user-hover" rel="olivergondza">olivergondza</a>, a normal nightly build of Firefox won't work given that it hasn't the patch from <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">bug 1370630</a> included. You can pick a build from <a href="https://treeherder.mozilla.org/#/jobs?repo=try&amp;revision=3f12e17b558a36ef5b65235d663c4e617c3d328e&amp;searchStr=build" class="external-link" target="_blank" rel="nofollow noopener">https://treeherder.mozilla.org/#/jobs?repo=try&amp;revision=3f12e17b558a36ef5b65235d663c4e617c3d328e&amp;searchStr=build</a> by just clicking the `B` symbol of your platform, selecting Job Details in the lower pane, and then `target.zip`, `target.dmg`, or `target.tar.bz2`. Just extract the archive and run the binary. With such a build you should be able to see the failure without the patch in Jenkins.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=olivergondza" class="user-hover" rel="olivergondza">olivergondza</a>, a normal nightly build of Firefox won't work given that it hasn't the patch from <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630" class="external-link" target="_blank" rel="nofollow noopener">bug 1370630</a> included. You can pick a build from <a href="https://treeherder.mozilla.org/#/jobs?repo=try&revision=3f12e17b558a36ef5b65235d663c4e617c3d328e&searchStr=build" class="external-link" target="_blank" rel="nofollow noopener">https://treeherder.mozilla.org/#/jobs?repo=try&revision=3f12e17b558a36ef5b65235d663c4e617c3d328e&searchStr=build</a> by just clicking the `B` symbol of your platform, selecting Job Details in the lower pane, and then `target.zip`, `target.dmg`, or `target.tar.bz2`. Just extract the archive and run the binary. With such a build you should be able to see the failure without the patch in Jenkins.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365305&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365305">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=olivergondza" class="user-hover" rel="olivergondza">olivergondza</a> AFAIUI, the behavior of Chrome is the correct one (i.e. future Firefox's), so you should be able to confirm using Chrome with a Firefox user agent string. That's how I reproduced it and worked on the fix.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=olivergondza" class="user-hover" rel="olivergondza">olivergondza</a> AFAIUI, the behavior of Chrome is the correct one (i.e. future Firefox's), so you should be able to confirm using Chrome with a Firefox user agent string. That's how I reproduced it and worked on the fix.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365306&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365306">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> Thanks for the link. "Save" buttons don't work on the nightly build and Jenkins 2.164.2, as expected. And they work on Jenkins 2.174, also as expected <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/biggrin.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> Thanks for the link. "Save" buttons don't work on the nightly build and Jenkins 2.164.2, as expected. And they work on Jenkins 2.174, also as expected <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/biggrin.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365312&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365312">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Great to be helpful. And that sounds fantastic. So we now have to wait for the 2.174 and an upcoming LTS before we can make that change live in Firefox.</p>
</pre>
</details>

</ul>
</details>

<p>Great to be helpful. And that sounds fantastic. So we now have to wait for the 2.174 and an upcoming LTS before we can make that change live in Firefox.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/olivergondza.png" title="olivergondza's avatar" /> olivergondza:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365315&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365315">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Cool, thanks! I am backporting this to 2.164.3 to be released <del>2019-04-08</del> (EDIT: 2019-05-08), unless further regressions are found.</p>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a>, I do appreciate the effort put in making sure Jenkins is ready for the fix in Firefox. Though I am wondering if there is a chance to postpone the delivery until significant Jenkins (or what other applications are affected) userbase migrate to the fixed version.</p>
</pre>
</details>

</ul>
</details>

<p>Cool, thanks! I am backporting this to 2.164.3 to be released <del>2019-04-08</del> (EDIT: 2019-05-08), unless further regressions are found.</p>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a>, I do appreciate the effort put in making sure Jenkins is ready for the fix in Firefox. Though I am wondering if there is a chance to postpone the delivery until significant Jenkins (or what other applications are affected) userbase migrate to the fixed version.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365316&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365316">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I assume you mean 2019-05-08. <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

<p>We can post-pone for sure. It's not that this patch is critical, and we already waited a while for it. Do you have some stats in how long it takes for the majority of users to upgrade to the next LTS release? Will it be one week, or two? Also note that the fix will land on mozilla-central only, which means it will only be part of a nightly build. Those people are tech-aware and should upgrade their instances fast enough.</p>

<p> </p>

<p>Our next merge to beta is on 2019-05-13, so I would suggest to at least wait until 2019-05-14 before landing that change in Firefox Nightly (69).</p>
</pre>
</details>

</ul>
</details>

<p>I assume you mean 2019-05-08. <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

<p>We can post-pone for sure. It's not that this patch is critical, and we already waited a while for it. Do you have some stats in how long it takes for the majority of users to upgrade to the next LTS release? Will it be one week, or two? Also note that the fix will land on mozilla-central only, which means it will only be part of a nightly build. Those people are tech-aware and should upgrade their instances fast enough.</p>

<p> </p>

<p>Our next merge to beta is on 2019-05-13, so I would suggest to at least wait until 2019-05-14 before landing that change in Firefox Nightly (69).</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365324&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365324">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> We are unfamiliar with the Firefox release schedule, could you clarify what that means in terms of rolling out the change to regular users? Those who are on 66.0.3 right now?</p>

<p>For Jenkins, based on usage statistics, 60% of LTS users are within the last 4 LTS releases (~4 months), and 50% of weekly release users are within the last 16 releases (~3.5 months). So I expect the majority of Jenkins users to be on compatible releases around August/September, and they <em>can</em> update no later than May (for LTS), and should already be updated (for weekly releases).</p>

<p>From a project POV, we only consider the latest LTS and weekly releases (e.g. only those get security fixes) to be supported. Of course users may lag behind in updating, and for some setups, updating Jenkins is a big project. The question the Firefox team will need to answer is, how aggressively do they want to annoy Firefox+Jenkins users that update one but not the other.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=whimboo" class="user-hover" rel="whimboo">whimboo</a> We are unfamiliar with the Firefox release schedule, could you clarify what that means in terms of rolling out the change to regular users? Those who are on 66.0.3 right now?</p>

<p>For Jenkins, based on usage statistics, 60% of LTS users are within the last 4 LTS releases (~4 months), and 50% of weekly release users are within the last 16 releases (~3.5 months). So I expect the majority of Jenkins users to be on compatible releases around August/September, and they <em>can</em> update no later than May (for LTS), and should already be updated (for weekly releases).</p>

<p>From a project POV, we only consider the latest LTS and weekly releases (e.g. only those get security fixes) to be supported. Of course users may lag behind in updating, and for some setups, updating Jenkins is a big project. The question the Firefox team will need to answer is, how aggressively do they want to annoy Firefox+Jenkins users that update one but not the other.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365330&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365330">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a> thank you for bringing this up. It's actually not my job to make such a decision, so I will leave it up to the release managers to decide. I will clearly reference your last comment, so that they are aware of the situation for Jenkins users.</p>

<p> </p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a> thank you for bringing this up. It's actually not my job to make such a decision, so I will leave it up to the release managers to decide. I will clearly reference your last comment, so that they are aware of the situation for Jenkins users.</p>

<p> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=365541&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-365541">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Ok, so here an update. If that patch sticks and will not be backed out again, we will land the Firefox patch for Firefox 69 on May 14th or 15th. Which means it will ride the trains through beta, and will finally be released on Sep 3rd. If by that time there are still users who running a Jenkins LTS release earlier than 2.164.3, they will have to use Firefox 68 ESR. That ESR version will be maintained for another year, and will never have this patch included. Details see <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630#c41" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630#c41</a> and following.</p>

<p> </p>

<p>Thanks a lot again for the patch! It will unblock us from landing several features which are all blocked by this particular issue.</p>
</pre>
</details>

</ul>
</details>

<p>Ok, so here an update. If that patch sticks and will not be backed out again, we will land the Firefox patch for Firefox 69 on May 14th or 15th. Which means it will ride the trains through beta, and will finally be released on Sep 3rd. If by that time there are still users who running a Jenkins LTS release earlier than 2.164.3, they will have to use Firefox 68 ESR. That ESR version will be maintained for another year, and will never have this patch included. Details see <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1370630#c41" class="external-link" target="_blank" rel="nofollow noopener">https://bugzilla.mozilla.org/show_bug.cgi?id=1370630#c41</a> and following.</p>

<p> </p>

<p>Thanks a lot again for the patch! It will unblock us from landing several features which are all blocked by this particular issue.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=379063&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-379063">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Reopened because 2.202 backed out the change again due to the regression it caused (<a href="https://issues.jenkins.io/browse/JENKINS-58296" title="Projects are triggered twice when using File Parameter and Firefox 67" class="issue-link" data-issue-key="JENKINS-58296"><del>JENKINS-58296</del></a>).</p>
</pre>
</details>

</ul>
</details>

<p>Reopened because 2.202 backed out the change again due to the regression it caused (<a href="https://issues.jenkins.io/browse/JENKINS-58296" title="Projects are triggered twice when using File Parameter and Firefox 67" class="issue-link" data-issue-key="JENKINS-58296"><del>JENKINS-58296</del></a>).</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=400603&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-400603">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hi <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>. After a year has been passed-by I wanted to check back if there has been made some progress on this particular issue. As known the change got backed out for 2.202, but was there any attempt again to get a fix landed?</p>

<p>On our side we would kinda like to remove this broken code from Firefox but are still blocked by this particular web-compat issue.</p>

<p>Thanks.</p>
</pre>
</details>

</ul>
</details>

<p>Hi <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>. After a year has been passed-by I wanted to check back if there has been made some progress on this particular issue. As known the change got backed out for 2.202, but was there any attempt again to get a fix landed?</p>

<p>On our side we would kinda like to remove this broken code from Firefox but are still blocked by this particular web-compat issue.</p>

<p>Thanks.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=400604&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-400604">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I'm confused, according to <a href="https://issues.jenkins-ci.org/browse/JENKINS-58296?focusedCommentId=376769&amp;page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-376769" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/JENKINS-58296?focusedCommentId=376769&amp;page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-376769</a> the Firefox change was backed out again, so I thought you'd given up on this.</p>
</pre>
</details>

</ul>
</details>

<p>I'm confused, according to <a href="https://issues.jenkins-ci.org/browse/JENKINS-58296?focusedCommentId=376769&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-376769" class="external-link" target="_blank" rel="nofollow noopener">https://issues.jenkins-ci.org/browse/JENKINS-58296?focusedCommentId=376769&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-376769</a> the Firefox change was backed out again, so I thought you'd given up on this.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=401042&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-401042">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>You are right, sorry. I will try to get a clarification from our DOM folks. I will report back when there are news.</p>
</pre>
</details>

</ul>
</details>

<p>You are right, sorry. I will try to get a clarification from our DOM folks. I will report back when there are news.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=407474&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-407474">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Fix attempt at <a href="https://github.com/jenkinsci/jenkins/pull/5405" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/5405</a></p>
</pre>
</details>

</ul>
</details>

<p>Fix attempt at <a href="https://github.com/jenkinsci/jenkins/pull/5405" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/5405</a></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/markewaite.png" title="markewaite's avatar" /> markewaite:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=409332&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-409332">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Change in Jenkins 2.289 seems to have caused a regression described in the <a href="https://github.com/jenkinsci/jenkins/pull/5405#issuecomment-842713911" class="external-link" target="_blank" rel="nofollow noopener">pull request comment</a>.</p>
</pre>
</details>

</ul>
</details>

<p>Change in Jenkins 2.289 seems to have caused a regression described in the <a href="https://github.com/jenkinsci/jenkins/pull/5405#issuecomment-842713911" class="external-link" target="_blank" rel="nofollow noopener">pull request comment</a>.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/whimboo.jpg" title="whimboo's avatar" /> whimboo:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-53462?focusedId=414058&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-414058">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, could you please give an update here? Given the last comment on this issue there is another regression. So far I haven't seen a backout, so does it mean your patch will be kept in the <a href="https://www.jenkins.io/changelog-stable/#v2.289.2" class="external-link" target="_blank" rel="nofollow noopener">https://www.jenkins.io/changelog-stable/#v2.289.2</a> LTS release? Thanks!</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck" class="user-hover" rel="danielbeck">danielbeck</a>, could you please give an update here? Given the last comment on this issue there is another regression. So far I haven't seen a backout, so does it mean your patch will be kept in the <a href="https://www.jenkins.io/changelog-stable/#v2.289.2" class="external-link" target="_blank" rel="nofollow noopener">https://www.jenkins.io/changelog-stable/#v2.289.2</a> LTS release? Thanks!</p>
